chart/values.yaml

```diff
@@ -1629,7 +1629,6 @@ statsd:
   uid: 65534
   # When not set, `statsd.uid` will be used
 
-  # When not set, the values defined in the global securityContext will be used
```

Comment:
Also need to remove this comment from values.schema.json.

Reply:
removed